### ISSUE TITLE
*20.0* for SM-A500G...

### ISSUE THREAD

**Issue by Hassan-kareem** [lwys-trash-oct23/lineage-18.1-a5lte-builds#3]
Hii.
Please give me the LOS20 build for Galaxy A5

**Comment by Dobby233Liu** [comment]
20.0 building ~~is broken by changes in device/lineage/sepolicy right now~~ I just vendored lineage sepolicy; even if I manage to fix it, the kernel version is still a blocking factor (IIRC Snow Cone and up requires eBPF which 3.x doesn't support) and idrk how to deal with it. In fact I'm thinking of abandoning 20.0 work and reverting the default branches to lineage-18.1 (**n.t.s.** still gonna do this)

**Comment by Dobby233Liu** [comment]
Wontfix then

**Comment by Hassan-kareem** [comment]
https://github.com/vknecht/msm8916-mainline-linux
It's not possible to use this kernel?
And any social media profile Sir like TG?

**Comment by Dobby233Liu** [comment]
> [vknecht/msm8916-mainline-linux](https://github.com/vknecht/msm8916-mainline-linux)

(**n.t.s.**) Got damn
I was only looking at the msm8916-mainline org didn't find this

> And any social media profile Sir like TG?

Only Discord is the easiest way to reach me rn, why though

**Comment by Hassan-kareem** [comment]
Never heard Discord before.
Actually I'm a user of Android, now I'm trying to learn some development

**Comment by Dobby233Liu** [comment]
> Actually I'm a user of Android, now I'm trying to learn some development

I don't think you will learn anything from me then :pray: I'm just an idiot who puts stuff together and expects them to work

**Comment by Hassan-kareem** [comment]
Once I was tried to build using your sources but error.
https://cirrus-ci.com/task/5073867706728448

**Comment by Dobby233Liu** [comment]
> Once I was tried to build using your sources but error.

You will need to make your build process use my patched qcom repos as you can see in [roomservice.xml](https://github.com/lwys-trash-oct23/local_manifests/blob/lineage-20.0/roomservice.xml). In this instance hardware/qcom-caf/wlan is to be replaced
(Edit: I see you do have the other extra repos in your [XML](https://github.com/Hassan-kareem/local_manifests/blob/lineage-20.0/roomservice.xml), however you still lack my wifi repo)

Im also obligated to mention that you will also need to apply [this patch](https://github.com/lwys-trash-oct23/lineage-18.1-a5lte-builds/blob/main/hardware-lineage-interfaces_camera-fix-build-failure-with-some-kernels.patch) to hardware/lineage/interfaces if you decide to build with temporarily the 3.x kernel, I know, it's terribly unobvious, and I should have just forked the repo instead (**n.t.s.** *uhh as in note to self*)

**Comment by Hassan-kareem** [comment]
Ok I'll try once if I can.
Btw thanks

**Comment by Hassan-kareem** [comment]
<p> Please build LOS 20 on ROM builder. Encountered error whenever I tried also using your <a href="https://cirrus-ci.com/task/6699924323041280">Manifest</a> </p>
